monero-outreach-docs/translations/es/PSA/files/teamwork/teamwork_es.sla

```diff
@@ -64,15 +67,44 @@
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="La primera parada cuando hablamos de la participación en los equipos, son los equipos de trabajo de la comunidad que se enfocan en organizar todos los demás equipos. Ellos te dirán si tu idea ya la desarrollaron y "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="con quienes hablar para llevarla a otro nivel. O si tu idea es completamente nueva, te conectarán con algunos recursos para facilitar que se haga una realidad."/>
+            <para/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Los desarrolladores núcleo (Core) de Monero nos han dado una cantidad de herramientas que nos permitan el éxito en los equipos de trabajo, incluyendo Mattermost para comunicación y Taiga para la planeación y administración de flujos de tareas para los proyectos. "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="Los individuos pueden participar de manera activa en discusiones que suceden en diferentes canales tales como Slack e IRC, diferentes grupos de chats en Telegram, "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="en Reddit vía r/Monero u otros subreddits dedicados "/>
+            <para/>
+            <ITEXT FONT="Adobe Caslon Pro Regular" FONTSIZE="11" FSHADE="90" CH="a las diferentes areas de Monero (como el minado y la especulación) y/o preguntando o respondiendo preguntas sobre Monero en Stack Exchange."/>
```

Comment:
"a las diferentes **áreas** de Monero...".

Reply:
also changed, "minado" to "minería".

Reply:
done